src/styles/wallpaper.module.scss

```diff
@@ -3,25 +3,12 @@
 .wallpaper {
   background-color: var(--bg-tertiary);
 
-  @include br.media-breakpoint-up(lg) {
```

Comment:
why do you prefer it without the wallpaper background on wide viewports?

It looks fine overall but I'd like to know more about the reasoning behind this decision.

Reply:
The main reason was that the contrast with the text color was too low and it was affecting readability.

I've also removed the max-width because it's something that doesn't fit well on a landing page IMHO:
The blog also looks nicer without the max width.

Reply:
👍